### PR TITLE
[FIX] Satisfy flake8 and pylint.

### PR DESCRIPTION
**Pull request by NL66278**
Try to solve lint issues

### CODE REVIEW

File: partner_relations/models/res_partner_relation.py [hunk 1]
```diff
@@ -48,44 +48,44 @@ def create(self, vals):
...
         return super(ResPartnerRelation, self).create(vals)
 
-    @api.one
     @api.constrains('date_start', 'date_end')
     def _check_dates(self):
         """End date should not be before start date, if not filled
 
         :raises exceptions.Warning: When constraint is violated
         """
+        self.ensure_one()
```
Comment (hbrunn) [inline partner_relations/models/res_partner_relation.py]: `ensure_one` simply raises if there's no or more than one record. If you actually want to get rid of the `@api.one` decorators, you need to rewrite the function to cope with recordsets. Especially constrains really need to be able to handle an arbitrary number of records. But the decorator has been de-deprecated to my knowledge (citation needed), so I'd ignore those warnings.

**Comment by NL66278** [comment]
Is this the citation you were looking for? https://github.com/odoo/odoo/issues/12408

**Comment by hbrunn** [comment]
looks about right, thanks!

**Comment by NL66278** [comment]
@hbrunn Restored @api.one.

**Comment by hbrunn** [comment]
thanks

**Pull request merged**